modules/gapi/include/opencv2/gapi/video.hpp

```diff
@@ -62,6 +62,12 @@ G_TYPED_KERNEL(GCalcOptFlowLKForPyr,
         return std::make_tuple(empty_array_desc(), empty_array_desc(), empty_array_desc());
     }
 };
+
+G_TYPED_KERNEL(GBackSubMOG2, <GMat(GMat)>, "org.opencv.video.backgroundSubtractormog2")
```

Comment:
I suggest to write the label with capital letters MOG - like "org.opencv.video.backgroundSubtractorMOG2"